src/main/java/vo/SplitTarget.java

```diff
@@ -0,0 +1,35 @@
+package vo;
+
+import java.util.Optional;
+
+import utils.StringUtils;
+
+public class SplitTarget {
+    private final String value;
+
+    private String delimiters = "[,:]";
```

Comment:
상수 이므로 상수 컨벤션을 따라가면 좋을 것 같습니다 😄 
```suggestion
    private static final String DELIMITERS = "[,:]";
```

Reply:
의도는 멤버 변수로 사용하는 것이었습니다.
 "//%s\n"에서 %s에 해당하는 값을 delimiters에 할당하는 로직이 있습니다. 
delimiters에 재할당하는 것이 개선이 필요한 패턴일까요?

Reply:
@aj172019 준헌님 

코드를 다시 보니 클래스 변수로 사용하신 값 이네요 🤔 
그럼 지금 구조가 맞습니다 😄 
컨벤션 부터 체크하다보니 그렇게 의견드린 것 같네요 🙏 
혼선드려 죄송합니다 🙇 

`split()` 호출시점에서 생성자를 통해 `delimiters`값이 결정이 되고 
중간에 다른 `method` 등에 의해  변질될 염려가 없으니 수정 안하셔도 될 것 같습니다 🙇 

다만 `extractDelimiters`만 보고 읽다보면 
`value`를 전달하고 `delimiters`를 반환 받는 모양처럼 보입니다만 (extract로 네이밍이 되어있어서), 
실제는 조건에 따라서 `delimiters` 값을 `re-assign`하게 되고, 실제 반환 되는 값은 `value`입니다. 
요 부분은 코드 읽기에 따라서 약간 혼동을 줄 수도 있을 것 같습니다 😄 
```java
private String extractDelimiters(String value) {
    if (value.startsWith("//") && value.contains("\n")) {
            delimiters = value.substring(2, value.indexOf("\n"));
            value = value.substring(value.indexOf("\n") + 1);
    }
    return value;
}

```